Sources/ScipioKit/Producer/PIF/ToolchainGenerator.swift

```diff
@@ -39,12 +42,24 @@ struct ToolchainGenerator {
         extraSwiftCFlags += ["-I", sdkPaths.lib.pathString]
         extraSwiftCFlags += ["-L", sdkPaths.lib.pathString]
 
+        #if swift(>=5.9)
+        let destination = Destination(
+            hostTriple: try? Triple("arm64-apple-\(sdk.settingValue)"),
+            targetTriple: try? Triple("arm64-apple-\(sdk.settingValue)"),
+            sdkRootDir: sdkPath,
+            toolchainBinDir: toolchainDirPath,
+            extraFlags: BuildFlags(cCompilerFlags: extraCCFlags, swiftCompilerFlags: extraSwiftCFlags)
+        )
+        return destination
+        #else
+
         return Destination(
             hostTriple: try? Triple("arm64-apple-\(sdk.settingValue)"),
             targetTriple: try? Triple("arm64-apple-\(sdk.settingValue)"),
             sdkRootDir: sdkPath,
             toolchainBinDir: toolchainDirPath,
             extraFlags: BuildFlags(cCompilerFlags: extraCCFlags, swiftCompilerFlags: extraSwiftCFlags)
         )
+        #endif
```

Comment:
What is changed here? Do we need to use variable in swift 5.9 here?

Reply:
🤔

Reply:
I re-checked that this was unnecessary. So I reverted.